src/scripts/plugins.es6

```diff
@@ -24,4 +24,4 @@
 			console[method] = noop
 		}
 	}
-})();
+})()
```

Comment:
Na řádek 8 bych předřadil středník, protože to je zrovna jediný případ, kdy je to dobrý nápad ho tam mít.